Mixpanel/Decide.swift

```diff
@@ -30,20 +30,20 @@ class Decide {
         }
     }
 
-    func checkDecide(forceFetch: Bool = false, distinctId: String, token: String, completion: ((response: DecideResponse?) -> Void)) {
+    func checkDecide(forceFetch: Bool = false, distinctId: String, token: String, completion: ((_ response: DecideResponse?) -> Void)) {
         var decideResponse = DecideResponse()
 
         if !decideFetched || forceFetch {
             let semaphore = DispatchSemaphore(value: 0)
             decideRequest.sendRequest(distinctId: distinctId, token: token) { decideResult in
                 guard let result = decideResult else {
                     semaphore.signal()
-                    completion(response: nil)
+                    completion(nil)
                     return
                 }
 
                 var parsedNotifications = [InAppNotification]()
-                if let rawNotifications = result["notifications"] as? [[String: AnyObject]] {
+                if let rawNotifications = result["notifications"] as? [[String: Any]] {
```

Comment:
Any allows Swift value types to be sent and not only object types. If it is AnyObject, it should work as well because the object is translated to an Objective-C object, but I would rather have it fully Swift :)